kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java

```diff
@@ -235,4 +283,43 @@ private JobRequestInstance createJobRequestInstance() {
         jobRequestInstance.setData(data);
         return jobRequestInstance;
     }
+    
+    private JobRequestInstance createJobRequestExecutionErrorInstance() {
```

Comment:
Could you parameterize this method for not being repeated passing the numberOfRetries?
```suggestion
    private JobRequestInstance createJobRequestExecutionErrorInstance(Integer numberOfRetries) {
```